controllers/whoToFollow/whoToFollowV2.js

```diff
@@ -214,6 +219,8 @@ module.exports = async (req, res) => {
                             AND a.karma_score > 30
                             AND a.user_id != :admin_user_id
                             AND b.topic_id NOT IN (:topics)
+                            AND a.user_id NOT IN (:allUserIds)
+                          GROUP BY a.user_id
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding a condition to exclude certain user IDs (`a.user_id NOT IN (:allUserIds)`) is a good way to avoid recommending the same users multiple times. However, be aware that using `NOT IN` with a large list of IDs could potentially slow down your query. If performance becomes an issue, consider other strategies such as storing already recommended users in a separate table and joining on that.



<!-- This is an auto-generated comment by OSS CodeRabbit -->